src/components/RadioButtons/RadioButtons.js

```diff
@@ -5,7 +5,7 @@ import useInvalid from '../../hooks/useInvalid.js'
 
 import { Body } from '../Type/Body.js'
 import { COLORS } from '../Colors.js'
-import styles from '../RadioButtons.module.scss'
+import styles from './RadioButtons.module.scss'
```

Comment:
Nick fixed this too lol...it'll FF

Reply:
eventually let's  node script / lint for these

Reply:
Lint PR (I wrote the code the other day, not in response to this comment, but why not open the PR?) https://github.com/getethos/ethos-design-system/pull/104